tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -277,8 +288,43 @@ <h3>No scalar data was found.</h3>
         return _smoothingWeight > 0;
       },
 
-      _shouldOpen(index) {
-        return index <= 2;
+      _getCategoryKey(category) {
+        return category.metadata.type ==
+            tf_categorization_utils.CategoryType.SEARCH_RESULTS ?
+                'search' : category.name;
+      },
+
+      _updatePaneOpened() {
+        const paneOpen = this._paneOpened;
+        const nextPaneOpen = {};
```

Comment:
Could you name this "new" instead of "next"?  "nextPane" sounds to me like it's literally the next pane down in the series of panes, but it actually is about "next" in time, not next in space.

Also, nit, but maybe "newPaneOpened" with the "ed" to match the actual property name.